tools/rv_histogram.c

```diff
@@ -231,21 +231,32 @@ static void insn_hist_incr(const rv_insn_t *ir)
 
 int main(int argc, const char *args[])
 {
+    int ret_val = 0;
+
     if (!parse_args(argc, args) || !elf_prog) {
         print_usage(args[0]);
-        return 1;
+        ret_val = 1;
+        goto end;
```

Comment:
Instead of using `goto`, you can use `return` instead.

Reply:
Alternatively, define a function named `fatal`, which dumps the message and invokes [exit](https://man7.org/linux/man-pages/man3/exit.3.html).

Reply:
You can safely ignore the warnings reported by Cppcheck at the moment. I shall suppress some of them.